packages/material/scss/listbox/_theme.scss

```diff
@@ -0,0 +1,26 @@
+@import "~@progress/kendo-theme-default/scss/listbox/theme";
+
+@include exports( 'listbox/theme/material') {
+    $listbox-item-focused-bg: rgba(0, 0, 0, .07) !default;
+    $listbox-drag-clue-bg: $hovered-bg !default;
+
+    .k-listbox {
+        .k-list .k-item {
+            color: $base-text;
+
+            &.k-state-focused {
+                background-color: $listbox-item-focused-bg;
```

Comment:
In the Default theme, `.k-list .k-item` gets its styles from the `popup/_theme.scss` file, which is inherited by the Material theme. Do we need the additional styles?